Resources/public/js/components/price-list/main.js

```diff
@@ -122,10 +130,23 @@ define([], function() {
             var bulkPriceComponents = [];
 
             this.sandbox.util.foreach(this.options.currencies, function(currency){
+
+                this.bulkPriceData = [];
```

Comment:
this can not work. As this is a list you can not add PriceListData.prices to it below. 
either you want a list and push a new item into it 

or you want to have an object ```{}``` and add a new property to it as you are currently doing it below